Source/Core/DolphinQt2/Config/Graphics/GraphicsSlider.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 Dolphin Emulator Project
```

Comment:
I wouldn't go so far as to say that those are bad criteria. But I disagree with how you interpret them. Even with your very detailed description of the possible advantages, none of it seems like it's a very big improvement even in the readability department  (or in some cases even no improvement at all, IMO).

In the end I don't want to spend my time changing functional and readable code that I - myself - think is perfectly serviceable. Not because I think I can claim some sort of huge objective merit (although I do think there is at least a small one) for that design decision, but because I think it's fine that way and perfectly harmless.

I could do a point by point response thingy, but I don't think that we are trying to figure out who is right on a mostly subjective matter and that would probably do no good.

I still do respect your opinion and thank you for taking the time to plough through my (sometimes massive) PRs.

Cheers.